Purchases.xcodeproj/project.pbxproj

```diff
@@ -1029,12 +1029,13 @@
 			path = Networking;
 			sourceTree = "<group>";
 		};
-		35F82BBB26A9BFA60051DF03 /* Misc */ = {
+		35F82BBB26A9BFA60051DF03 /* FoundationExtensions */ = {
```

Comment:
moved stuff around: 
- renamed `Misc` to `FoundationExtensions` to better represent the contents
- moved the new `DateFormatter` extension to `FoundationExtensions`